z3c/dependencychecker/package.py

```diff
@@ -176,3 +177,21 @@ def _get_sources_top_level(self):
             os.sep,
         )
         sys.exit(1)
+
+
+class Package(object):
+
+    def __init__(self, path):
+        self.metadata = PackageMetadata(path)
+        self.imports = ImportsDatabase()
+
+    def get_declared_dependencies(self):
+        """Get this packages' dependencies defined in setup.py"""
+        self.imports.set_requirements(
+            self.metadata.get_required_dependencies()
+        )
+
+    def get_declared_extras_dependencies(self):
```

Comment:
This is a getter, but you're calling a setter inside it? (set_extra_requirements)?
Looks weird (or looks like a typo).

Reply:
Right, I renamed both methods to ``set_xxxx``